.github/CONTRIBUTING.md

```diff
@@ -463,6 +463,57 @@ When adding new signals to root level procs, eg;
 ```
 The `SHOULD_CALL_PARENT(TRUE)` lint should be added to ensure that overrides/child procs call the parent chain and ensure the signal is sent.
 
+### Use descriptive and obvious names
+Optimize for readability, not writability. While it is certainly easier to write `M` than `victim`, it will cause issues down the line for other developers to figure out what exactly your code is doing.
```

Comment:
A lot of people seem to fall into the trap of thinking 'because the variable is only used for one line, it's super easy to figure out its purpose without a descriptive name'